benchmarks/queries/q20.sql

```diff
@@ -28,7 +28,7 @@ where
                     l_partkey = ps_partkey
               and l_suppkey = ps_suppkey
               and l_shipdate >= date '1994-01-01'
-              and l_shipdate < 'date 1994-01-01' + interval '1' year
+              and l_shipdate < date '1994-01-01' + interval '1' year
```

Comment:
I don't understand what changed here 🤷

Reply:
There was a syntax error regarding placement of the quotation mark.